converters/fromZigbee.js

```diff
@@ -130,15 +130,17 @@ const converters = {
             const value = msg.data.data['presentValue'];
             let action = null;
 
-            if (value === 0) action = 'shake';
-            else if (value === 2) action = 'wakeup';
-            else if (value === 3) action = 'fall';
-            else if (value >= 512) action = 'tap';
-            else if (value >= 256) action = 'slide';
-            else if (value >= 128) action = 'flip180';
-            else if (value >= 64) action = 'flip90';
+            if (value === 0) action = {'action': 'shake'};
+            else if (value === 2) action = {'action': 'wakeup'};
+            else if (value === 3) action = {'action': 'fall'};
+            else if (value >= 512) action = {'action': 'tap', 'side': value-512};
+            else if (value >= 256) action = {'action': 'slide', 'side': value-256};
+            else if (value >= 128) action = {'action': 'flip180', 'side': value-128};
```

Comment:
If I understand correctly`side` is a number, what does this mean?

Reply:
Yes, it is number. Sides numered as picture (from file):
```
                +---+
                | 2 |
            +---+---+---+
            | 4 | 0 | 1 |
            +---+---+---+
                |M5I|
                +---+
                | 3 |
                +---+
```